src/mjolnir/pbfgraphparser.cc

```diff
@@ -1828,18 +1830,6 @@ struct graph_callback : public OSMPBF::Callback {
     bss_nodes_.reset(bss_nodes);
   }
 
-  // Output list of wayids that have loops
-  void output_loops() {
```

Comment:
i removed this because its no longer useful now that the code successfully handles loops. we can let other projects more suited to data cleanup handle calling out degeneracies in the data